subsys/zigbee/Kconfig

```diff
@@ -171,6 +171,31 @@ endmenu #menu "Zigbee Log configuration"
 
 menu "ZBOSS osif configuration"
 
+menuconfig ZIGBEE_HAVE_SERIAL
+	bool "Enable UART serial interface for ZBOSS CLI"
+	select SERIAL
+	select UART_ASYNC_API
+
+if ZIGBEE_HAVE_SERIAL
+
+config ZIGBEE_UART_DEVICE_NAME
+	string "Zigbee UART device name"
+	default "UART_1"
+
+config ZIGBEE_UART_TX_TIMEOUT
+	int "Timeout value for starting asynchronous transmission"
```

Comment:
From user perspective, would be good to provide an unit for this value (seconds? miliseconds?).

Reply:
Fixed.